packages/vuetify/src/components/VCalendar/VCalendarCategory.ts

```diff
@@ -30,12 +30,21 @@ export default VCalendarDaily.extend({
       return typeof this.categories === 'string' && this.categories
         ? this.categories.split(/\s*,\s*/)
         : Array.isArray(this.categories)
-          ? this.categories as string[]
+          ? this.categories.map((v: any) => {
```

Comment:
code is duplicated in Calendar.ts, could be extracted to an util along with parseCategoryText